mpa/modules/hooks/checkpoint_hook.py

```diff
@@ -53,14 +53,15 @@ def after_train_epoch(self, runner):
             return
         if hasattr(runner, 'save_ckpt'):
             if runner.save_ckpt:
-                if runner.save_ema_model:
+                # TODO: Do we need this even after we moved from torchreid to mmcls?
```

Comment:
MPA classification now also supports EMA model

Reply:
Removed `todo` in https://github.com/openvinotoolkit/model_preparation_algorithm/pull/106/commits/352d48aa9a102ad2526cf2b0967fc5dc1b07f180